app/services/authorizer.rb

```diff
@@ -105,6 +105,12 @@ def build_scoped_search_condition(filters)
     raise ArgumentError if filters.blank?
 
     strings = filters.map { |f| "(#{f.search_condition.presence || '1=1'})" }
+    # strings = []
+    # filters.each do |f|
+    #   next if f.search_condition.blank?
+    #   strings << "(#{f.search_condition})"
+    # end
```

Comment:
This seems to do the trick, but I'm wondering if it could be rewritten in a more functional style without mutating data in place?

Reply:
yeah, it make sense, but I'm wondering how comes we even ever hit this case... :thinking: